pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -313,7 +313,12 @@ func (r *Builder) DataVolumes(vmRef ref.Ref, secret *core.Secret, _ *core.Config
 			if disk.Datastore.ID == ds.ID {
 				storageClass := mapped.Destination.StorageClass
 				var dvSource cdi.DataVolumeSource
-				if r.Plan.Spec.Warm {
+				if r.Destination.Provider.IsHost() && !r.Plan.Spec.Warm {
```

Comment:
It would be best to use `r.Context.UseEl9VirtV2v()` here.

Reply:
indeed